Neuro/Recording/DataStructures/SabotageTypes.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Neuro.Recording.DataStructures
+{
+    public enum SabotageTypes
```

Comment:
I was using SystemTypes at first, but I figured it would be kinda confusing for this to be random numbers since the sabotages are spread out over the enum